reddit/reddit.py

```diff
@@ -67,63 +77,60 @@ def top_25_scanner():
+
+    return filtered_submissions
 
 
 def create_submission_json(submissions: list, subreddit):
-    path = "../data_collected/reddit/" + subreddit
-    file = str(time.time()) + ".json"
-    Path(path).mkdir(parents=True, exist_ok=True)
-
-    with open(os.path.join(path, file), "w") as f:
-        json.dump(submissions, f)
-
-    if cli_arguments:
-        # TODO: Has to be modified for "STM-114:dynamic output directories" later on
-        with open("../data_collected/reddit/" + subreddit + "/" + file) as f:
-            data = json.load(f)
-            stuff = find_stock(sys.argv[1:])
-            for key, value in stuff.items():
-                filtered_json = []
-                for submission in data:
-                    relevant_comments = [
-                        # Will look for a complete ticker match within the comment body, or
-                        # Will look for a word match within the body.
-                        # For example, "Tesla Inc. Common Stock" and "I love stocks" will return a match,
-                        # whereas "Tesla Inc. Common Stock" and "I love Tes" (should) not.
-
-                        # TODO: Potentially look into removing common words such as "stock" from nasdaq.csv and only keep company name
-                        comment for comment in submission['comments'] if key.lower() in comment['body'].lower() or
-                                                                         any(re.search(r"\b" + re.escape(word.lower()) +
-                                                                                       r"\b", comment['body'].lower())
-                                                                             for word in value.split())
-                    ]
-                    # If we find any matches, replace the old list in the submission object with the new filtered list
-                    # We build a new JSON using the old one with only filtered comments as a result
-                    if relevant_comments:
-                        filtered_submission = submission.copy()
-                        filtered_submission['comments'] = relevant_comments
-                        filtered_json.append(filtered_submission)
-
-                #TODO: As is, we will most likely have duplicate comments since a new sub-directory is being made for every
-                # cli arg. Could just do one output for all args, but then it would be difficult to tell which cli arg
-                # comment had a match for. Discuss with team later.
-
-                filter_path = "../data_collected/reddit/" + subreddit + "/" + key
-                Path(filter_path).mkdir(parents=True, exist_ok=True)
-                file = str(time.time()) + ".json"
-                with open(os.path.join(filter_path, file), "w") as filtered:
-                    json.dump(filtered_json, filtered)
-
-
-top_25_scanner()
+    try:
+        path = "../data_collected/reddit/" + subreddit
```

Comment:
Instead of hard-coding this, let's make it an env variable. If the variable isn't set, use this path by default